app/src/main/java/mozilla/lockbox/store/SettingStore.kt

```diff
@@ -26,13 +27,19 @@ open class SettingStore(
     object Keys {
         const val SEND_USAGE_DATA = "send_usage_data"
         const val ITEM_LIST_SORT_ORDER = "sort_order"
+        const val UNLOCK_WITH_FINGERPRINT = "unlock_with_fingerprint"
     }
 
     private lateinit var preferences: SharedPreferences
     private val compositeDisposable = CompositeDisposable()
 
     open lateinit var sendUsageData: Observable<Boolean>
     open lateinit var itemListSortOrder: Observable<ItemListSort>
+    open lateinit var unlockWithFingerprint: Observable<Boolean>
+
+    open val onAuthentication: Observable<FingerprintAuthAction> =
```

Comment:
Does this fingerprint authentication belong in `SettingStore` or `FingerprintStore`? It seems specific to the Fingerprint, not the Settings.

Reply:
This is for enabling fingerprint, but in fact it is still an authentication. I will change the naming to be more specific